protocol/use-cases.md

```diff
@@ -20,14 +20,13 @@ This means OptyFi users would have access to yield-generating opportunities acro
 
 ### Vault Boosting
 
-OptyFi’s Vault Booster smart contract enables any user/contract to incentivize any ecosystem vault by submitting a transaction that includes the appropriate number of reward tokens along with the rate at which those reward tokens are to be distributed to the target vault. This enables project teams and contracts to influence flow of capital on demand and presents a myriad of possible use cases such as:
+OptyFi’s Vault Booster smart contract enables any user/contract to incentivize any ecosystem vault by submitting a transaction that includes the appropriate number of reward tokens along with the rate at which those reward tokens are to be distributed to the target vault. This enables project teams and contracts to influence flow of capital on demand and present a myriad of possible use cases such as:
 
 * Flexible mechanisms for DAOs to incentivize other DAOs.
 * Providing “venture capital” to build a target project's liquidity by attracting liquidity providers.
 
 ### Factor-Based and Thematic Yield Investing
 
-By explicitly defining the universe of discrete yield strategies, OptyFi enables application of the principles of factor-based investing and thematic investing to yield generation. Thus, instead of only trying to optimize for APY, strategists can optimize strategies based on exposure to various factors and yield portfolios can be developed that specify weight exposures to specific factors. Of course risk could be a primary factor, but almost any factor can be defined such as exposure to pool type \(e.g. DEX, Lending, Derivatives\) or exposure to protocol age. Similarly, vaults can be configured to invest thematically, for example to invest only in liquidity pools related to Ethereum L2 scaling solutions.
+By explicitly defining an assortment of discrete yield strategies, OptyFi enables application of the principles of factor-based investing and thematic investing to yield generation. Thus, instead of only trying to optimize for APY, strategists can optimize strategies based on exposure to various factors and yield portfolios can be developed that specify weight exposures to specific factors. Theoretically, all factors can be utilized to determine a strategy, such as exposure to pool type \(e.g. DEX, Lending, Derivatives\) or exposure to protocol age. Similarly, vaults can be configured to invest thematically, for example, by investing only in liquidity pools related to Ethereum L2 scaling solutions.
```

Comment:
Recommendation:
-...explicitly defining an all-inclusive ensemble of ...
-...exposure to various factors, *thus* yield
-Theoretically, almost any factor can....

Change:
-"By investing" ---> investment only....

Reply:
This change:
-"By investing" ---> investment only....
would result in an incorrect sentence for me. I have left it as is for now.